saleor/discount/migrations/0077_unique_type.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.24 on 2024-03-26 08:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("checkout", "0065_checkout_tax_error"),
+        ("order", "0183_order_tax_error"),
+        ("discount", "0076_promotionrule_variants_dirty"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="checkoutdiscount",
+            name="unique_type",
+            field=models.CharField(blank=True, max_length=64, null=True),
+        ),
+        migrations.AddField(
+            model_name="checkoutlinediscount",
+            name="unique_type",
+            field=models.CharField(blank=True, max_length=64, null=True),
+        ),
+        migrations.AddField(
+            model_name="orderdiscount",
+            name="unique_type",
+            field=models.CharField(blank=True, max_length=64, null=True),
+        ),
+        migrations.AddField(
+            model_name="orderlinediscount",
+            name="unique_type",
+            field=models.CharField(blank=True, max_length=64, null=True),
+        ),
+        migrations.AlterUniqueTogether(
+            name="checkoutdiscount",
+            unique_together={
+                ("checkout_id", "unique_type"),
+                ("checkout_id", "promotion_rule_id"),
+            },
+        ),
+        migrations.AlterUniqueTogether(
```

Comment:
we need to add indexes that will be created for uniqueness concurrently.

Reply:
According to postgres docs, it is created automatically:
https://www.postgresql.org/docs/9.4/indexes-unique.html#:~:text=PostgreSQL%20automatically%20creates%20a%20unique%20index%20when%20a%20unique%20constraint%20or%20primary%20key%20is%20defined%20for%20a%20table.%20The%20index%20covers%20the%20columns%20that%20make%20up%20the%20primary%20key%20or%20unique%20constraint%20(a%20multicolumn%20index%2C%20if%20appropriate)%2C%20and%20is%20the%20mechanism%20that%20enforces%20the%20constraint.

Reply:
Got your point, we need to ensure it is added **CONCURRENTLY**